test/functional/FormatSpec.hs

```diff
@@ -90,15 +90,15 @@ spec = do
       doc <- openDoc "BrittanyLF.hs" "haskell"
       sendNotification WorkspaceDidChangeConfiguration (DidChangeConfigurationParams (formatLspConfig "brittany"))
       let opts = DocumentFormattingParams doc (FormattingOptions 4 True) Nothing
-      ResponseMessage _ _ (Just edits) _ <- request TextDocumentFormatting opts
+      ResponseMessage _ _ (Right (Just edits)) <- request TextDocumentFormatting opts
       liftIO $ edits `shouldBe` [TextEdit (Range (Position 0 0) (Position 5 0))
                                   "module BrittanyLF where\n\nfoo :: Int -> String -> IO ()\nfoo x y = do\n    print x\n    return ()\n"]
 
     it "formats a document with CRLF endings" $ runSession hieCommand fullCaps "test/testdata" $ do
       doc <- openDoc "BrittanyCRLF.hs" "haskell"
       sendNotification WorkspaceDidChangeConfiguration (DidChangeConfigurationParams (formatLspConfig "brittany"))
       let opts = DocumentFormattingParams doc (FormattingOptions 4 True) Nothing
-      ResponseMessage _ _ (Just edits) _ <- request TextDocumentFormatting opts
+      ResponseMessage _ _ (Right (Just edits)) <- request TextDocumentFormatting opts
```

Comment:
I am not sure if these updates should be just to `(Right edits)`.  The functional tests pass locally with it this way.

Reply:
Yeah that should be correct, response messages merged the error field and result field into an `Either`, where `Right` is a result